proto/substrait/plan.proto

```diff
@@ -46,6 +46,10 @@ message Plan {
   // unused. In many cases, a consumer may be able to work with a plan even if
   // one or more message types defined here are unknown.
   repeated string expected_type_urls = 5;
+
+  // An optional list of bindings for dynamic parameters used in this plan.
+  // Each binding maps a parameter_anchor to its corresponding runtime value.
+  repeated ParameterBinding parameter_bindings = 7;
```

Comment:
If this was `repeated Expression.Literal parameter_bindings = 7;` then the anchor could just be the parameter's position in the list.

Reply:
Yes, I had it that way originally, then I changed it to make it consistent with other anchors such as the functions. I'll change it back.